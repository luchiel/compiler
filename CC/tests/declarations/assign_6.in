void main()
{
int a;
double b;
b = a % 7;
}