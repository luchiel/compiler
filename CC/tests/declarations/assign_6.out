type <int>
type <double>
type <void>
type <int*>
	pointer to <int>
type <void*>
	pointer to <void>
variable NULL of type <void*> initialized by
	{int 0}
		type <int>
function printf
	returns type <int>
	arguments
		<no args>
function main
	returns type <void>
	arguments
		<no args>
	{{}}
		variable a of type <int>
		variable b of type <double>
	| 
	+-{;}
	  | 
	  +-{=}
				type <double>
	    | 
	    +-{b}
					type <double>
	    | 
	    +-{cast}
					type <double>
	      | 
	      +-<double>
	      | 
	      +-{%}
						type <int>
	        | 
	        +-{a}
							type <int>
	        | 
	        +-{int 7}
							type <int>
